matrix/mat64/shadow.go

```diff
@@ -208,7 +208,7 @@ func rectanglesOverlap(off, aCols, bCols, stride int) bool {
 
 	// Flatten the shifted matrix column positions
 	// so a starts at 0, modulo the common stride.
-	const aFrom = 0
+	// const aFrom = 0
```

Comment:
I should have left the const in the code with the hope the compiler optimised it away. It will take time to figure out where it actually sits, so maybe just removing it is OK.